airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/Source.java

```diff
@@ -36,4 +37,14 @@ public interface Source extends Integration {
    */
   AutoCloseableIterator<AirbyteMessage> read(JsonNode config, ConfiguredAirbyteCatalog catalog, JsonNode state) throws Exception;
 
+  /***
+   * This method returns connector name which user for error messages mapping
+   *
+   * @return ConnectorName for example MYSQL or POSTGRES, will return DEFAULT if error message mapping
+   *         is not specified
+   */
+  default ConnectorName getConnectorName() {
+    return ConnectorName.DEFAULT;
```

Comment:
i'm not sure that it's right way to return DEFAULT instead of true connectors name. I would prefer return  MYSQL or POSTGRES, etc